src/commands/synthetics/command.ts

```diff
@@ -265,6 +267,7 @@ RunTestCommand.addOption('failOnMissingTests', Command.Boolean('--failOnMissingT
 RunTestCommand.addOption('failOnTimeout', Command.Boolean('--failOnTimeout'))
 RunTestCommand.addOption('files', Command.Array('-f,--files'))
 RunTestCommand.addOption('jUnitReport', Command.String('-j,--jUnitReport'))
+RunTestCommand.addOption('mobileApplicationVersionFilePath', Command.String('--mobileApplicationVersionFilePath'))
```

Comment:
Should we add an option shortcut? All the single letters switches I can think of are already used, so it could be just a shorter option name...

What about `--mobileApplication`?

Reply:
Maybe even shorter like `--mobileApp`